### ISSUE TITLE
Android: Sorting selected images by selection order

### ISSUE THREAD

**Issue by jonasfunk** [prashantsaini1/titanium-android-imagepicker#19]
Images returned by imagepicker is returned by creation date. Is it possible to sort by selection order?

**Comment by prashantsaini1** [comment]
I believe it's already returning in selection-order. Are you sure they are not?

**Comment by jonasfunk** [comment]
No, unfortunately they return in the order of placement in gallery. I have fixed this locally. Will submit PR :)

Sendt fra min iPhone

**Comment by emeisinga** [comment]
I am also experiencing this problem, could this be fixed? It will solve my problem.

**Comment by emeisinga** [comment]
@jonasfunk did you submitted the fix?
